### DISCUSSION TITLE
E492: Not an editor command: LazyExtras

### DISCUSSION THREAD

**Discussion by fabricetriboix** [LazyVim/LazyVim#3118] [Q&A] [answered]
Hello everybody,

I installed nvim using the latest AppImage, and then I install NvChad by following the [official instructions](https://nvchad.com/docs/quickstart/install/).

When I try to access Lazy extras by running `:LazyExtras` in nvim, it fails with the follow error:

E492: Not an editor command: LazyExtras

What am I doing wrong?

Thanks for your help!

**Comment by dpetka2001** [comment]
`LazyExtras` is only for LazyVim as far as I know. But you say that you installed NvChad.

**Reply by fabricetriboix** [reply]
Thanks for the super-quick answer @dpetka2001 . I am very new to neovim. What should I do to be able to install the Lazy extra packages in my situation?

**Reply by dpetka2001** [reply, marked as answer]
You can't. You should either uninstall NvChad and install LazyVim as per the [docs](https://www.lazyvim.org/installation) and be able to use `LazyExtras` or keep NvChad and not be able to use that command.

**Reply by fabricetriboix** [reply]
All right, thanks @dpetka2001
